hugo/content/privacy/_index.md

```diff
@@ -22,6 +19,12 @@ We want to inform you that whenever you visit our **Service**, we collect the in
 The personal information that we collect is used for providing and improving the **Service**.
 We will not use or share your information with anyone except as described in this policy. All collected personal information will be processed for research purposes, i.e. using the lawful basis of public interest and in accordance with Regulation (EU) 2016/679 of the European Parliament and of the Council of 27 April 2016, the General Data Protection Regulation.
 
+### Contact Form
+
+Our **Service** contains a <a href="/contact" target="_blank">contact form</a> that visitors can use to contact us or send us suggestions. The website visitors may choose to provide their personal information such as their name and e-mail address through this form.
+
+The following parties will have access to processing the personal data provided through the form; SciLifeLab Data Centre, Uppsala University, Kungliga Tekniska högskolan (KTH) and Naturhistoriska riksmuseet (NRM). Your personal data will be deleted when no longer needed, or when stipulated by the archival rules for the university as a government authority. If you want to update or remove your personal data, please contact the controller SciLifeLab Data Centre at Uppsala University using [dsn-eb@scilifelab.se](mailto:dsn-eb@scilifelab.se).
```

Comment:
Should we perhaps use the English names for all institutions and not just for UU? My understanding is that their official English versions are: `KTH Royal Institute of Technology, and Swedish Museum of Natural History`. 

(I suspect that there might be inconsistencies about this in other pages in the portal too :smile: )

Reply:
Fair point, I've now made them all use their English names in commit https://github.com/ScilifelabDataCentre/genome-portal/pull/57/commits/6df7a70e09aa2bde6268ef1d1c71d38993838d3f